source/wp-content/themes/wporg-showcase-2022/parts/header.html

```diff
@@ -1,2 +1,2 @@
 <!-- wp:wporg/global-header /-->
-<!-- wp:pattern {"slug":"wporg-parent-2021/subnav-bar"} /-->
+<!-- wp:pattern {"slug":"wporg-showcase-2022/subnav"} /-->
```

Comment:
What do you think about keeping this as `/subnav-bar`? Mostly as a devex thing, so we know it's meant to be a fork of the parent's pattern.

Reply:
Yep. Fine by me.